### PR TITLE
Cleanup of inactive contributors from OWNERS files

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: contributors/guide/owners.md
Comment (SergeyKanzhelev) [inline contributors/guide/owners.md:138]:
```suggestion
- PR comments are less than 10 and Devstats count is less than 10 for a year
```

Or alternatively  say "year" on top for both items.

File: contributors/guide/owners.md
Comment (SergeyKanzhelev) [inline contributors/guide/owners.md:121]:
```suggestion
prune folks from the OWNERS files who are not active in the community or specific repository. This
```

We may want to split some repos, especially for reviewers. For example, sig node approvers may be approvers in test-infra sig node areas, but not necessarily a reviewers there.

File: contributors/guide/owners.md
Comment (SergeyKanzhelev) [inline contributors/guide/owners.md:118]: what may help is to make reviewers list more dynamic, while applying very conservative metrics to approvers. It may also help if bot automation will suggest reviewers before suggesting approvers. Today PRs are saying to assign to the approver and suggest the approver. Also reviewers stats might be split by repos. 

wdyt?

Comment (mrbobbytables) [reply contributors/guide/owners.md:118]: The reviewers that are assigned are from the reviewer pool, ideally the workflow pattern is: Reviewer assigned to PR reviews and applies lgtm and/or assigns the approver. If they did not assign and just applied lgtm then the PR author assigns the approvers listed.

File: contributors/guide/owners.md
Comment (mrbobbytables) [inline contributors/guide/owners.md:137]:
```suggestion
An inactive OWNER is defined as someone with less than 10 Devstats recorded
contributions within the past year, as shown by this [dashboard]. This is a
conservative metric but should ensure only the removal of the most inactive
OWNERS.
```

Please convert the dashboard link to a reference style link. (They're being used throughout the doc)

File: contributors/guide/owners.md
Comment (mrbobbytables) [reply contributors/guide/owners.md:138]:
```suggestion
```
(GitHub would not let me do a suggestion over this whole block)
The Devstats dashboard being referenced is just contributions, if we're using that as the source of truth I think this line can be removed.

File: contributors/guide/owners.md
Comment (mrbobbytables) [inline contributors/guide/owners.md:144]:
```suggestion
```
If the earlier suggestions are accepted. 👍

File: contributors/guide/owners.md
Comment (dims) [reply contributors/guide/owners.md:118]: if there's a todo from this convo, please open an issue and we can deal with it there. thanks!

**Comment by dims** [comment]
@SergeyKanzhelev @mrbobbytables please revisit. i've accepted all the suggestions.

File: contributors/guide/owners.md
Comment (mrbobbytables) [inline contributors/guide/owners.md:129]: Just need to put a link to the dashboard down below 👍

Comment (dims) [reply contributors/guide/owners.md:129]: Done. thanks @mrbobbytables

**Review by idvoretskyi — APPROVED** [review]
/lgtm

/hold for other reviewers

**Comment by dims** [comment]
related to https://github.com/kubernetes/kubernetes/pull/106436/

**Review by mrbobbytables — APPROVED** [review]
/lgtm

**Review by justaugustus — APPROVED** [review]
/lgtm
/approve
/hold cancel

File: contributors/guide/owners.md
Comment (smarterclayton) [inline contributors/guide/owners.md:132]: So reading this I'm a bit confused - is an inactive OWNER moved to `emeritus_approvers`?  Or simply removed?  Looks like the automation moves them to emeritus?  If that's the case, then we should describe it with a sentence in this section ("if you are made emeritus here's where it goes")

Also, in the list of "valid fields" described for `OWNERS` here https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners we need to list emeritus approvers and link below.  I read the doc while reviewing felt that would be the right place to introduce emeritus (since now emeritus is more prevalent due to automation, and the likelihood someone wants to go back to approvers by contributing again).

Comment (smarterclayton) [reply contributors/guide/owners.md:132]: Also, I feel like the OWNERS file description should probably split "fields that list people" and "all other fields (i.e. `options` / `labels`)".  It's a key part of the process, and we shouldn't miss the opportunity to explain.  I can try to apply some feedback after the other stuff matures.

Comment (dims) [reply contributors/guide/owners.md:132]: fixing in https://github.com/kubernetes/community/pull/6339

**Review by Mxdon123 — APPROVED** [review]

**Review by Mxdon123 — APPROVED** [review]

**Pull request merged**
